src/services/bonusApiClientFactory.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * This service builds Bonus API clients.
+ */
+
+import nodeFetch from "node-fetch";
+
+import { BonusAPIClient } from "../clients/bonus";
+import { IBonusAPIClientFactoryInterface } from "./IBonusApiClientFactory";
+
+// TODO: this class is actually useless as BonusAPIClient is immutable, it can be removed
```

Comment:
exactly, get rid of this class / factory which is useless. we'll call new BonusApiClient() when needed

Reply:
should we do the same for `ApiClientFactory` and `PagoPAClientFactory`? They are actually the same (except pagopa creates a different client based on the environment, but such decision can be made in `config.ts` imho)

Reply:
yes but not with this PR :) very low priority

Reply:
tracked it [here](https://www.pivotaltracker.com/story/show/173075418)